src/appleseedmaya/shadingnode.cpp

```diff
@@ -122,15 +122,31 @@ namespace
             attr.setNiceNameOverride(p.label);
 
         if (p.mayaAttributeConnectable == false)
+        {
             attr.setConnectable(false);
+            attr.setStorable(true);
```

Comment:
We already set these properties as part of AttributeUtils::makeInput and AttributeUtils::makeOutput.
If we miss any, it would be better to add them in those functions.